src/main/asciidoc/index.adoc

```diff
@@ -164,20 +164,17 @@ Because of asynchronous operations it is possible that the effects of these meth
 
 This is a problem of JUnit 5 rather than this module. In case of doubt you may always wonder why a single method can't be better than many.
 
-==== Support for RxJava 1 and 2 Vertx instances
+== Support for additional parameter types
```

Comment:
Can you add links to the docs of the extensions? (Given that we won't have them in the main vertx.io/docs page)

Reply:
Let's wait for the PR to be merged and the other repository to be filled.